api/v3/examples/Setting/GetFields.php

```diff
@@ -479,6 +479,21 @@ function setting_getfields_expectedresult() {
         'description' => 'If enabled, CiviCRM will permit submissions from external sites to profiles. This is disabled by default to limit abuse.',
         'help_text' => '',
       ),
+      'dismiss_info_alerts' => array(
+        'group_name' => 'CiviCRM Preferences',
+        'group' => 'core',
+        'name' => 'dismiss_info_alerts',
+        'type' => 'Boolean',
+        'quick_form_type' => 'YesNo',
+        'default' => '1',
+        'html_type' => 'radio',
+        'add' => '4.7',
+        'title' => 'Dismiss info alerts',
+        'is_domain' => 1,
+        'is_contact' => 0,
+        'description' => 'If disabled, CiviCRM will stop dismissing the information alerts.',
```

Comment:
How about, 'If disabled, alert messages in CiviCRM will never be automatically dismissed after 10 seconds.'